daemon/algod/api/server/v2/test/helpers.go

```diff
@@ -168,7 +168,7 @@ func (m *mockNode) BroadcastSignedTxGroup(txgroup []transactions.SignedTxn) erro
 }
 
 func (m *mockNode) Simulate(request simulation.Request) (simulation.Result, error) {
-	simulator := simulation.MakeSimulator(m.ledger.(*data.Ledger))
+	simulator := simulation.MakeSimulator(m.ledger.(*data.Ledger), m.config)
```

Comment:
I'd prefer to see us pass the exact stuff down we need. If we pass `config` we're going to end up throwing things in there willy-nilly.